RecoTracker/Record/BuildFile.xml

```diff
@@ -9,3 +9,6 @@
 <use name="CalibTracker/Records"/>
 <use name="clhep"/>
 <use name="boost"/>
+<use name="alpaka"/>
+<flags ALPAKA_BACKENDS="1"/>
+
```

Comment:
The `RecoTracker/Record` package seems to be intended for EventSetup Record classes. I'd suggest to keep it that way, i.e. to move the new SoA definitions elsewhere.